content/sensu-go/5.10/guides/reduce-alert-fatigue.md

```diff
@@ -32,6 +32,10 @@ place, learn [how to send alerts with handlers][3].
 
 ### Creating the filter
 
+We'll show you two approaches to creating a filter that will handle occurrences. The first approach will be to creat our own filter that we'll add to Sensu. The second approach will cover implementing the filter as an asset.
+
+#### Using Sensuctl to Create a Filter
```

Comment:
Because this is an h4 heading, the resulting structure of the main section of the guide is:

- Using filters to reduce alert fatigue
	- Creating the filter
		- Using Sensuctl to Create a Filter
	- Assigning the filter to a handler
		- Using a Filter Asset
		- Annotating a check for filter asset use
		- Assigning the filter to a handler
	- Validating the filter

To ensure a clear separate between the two approaches, I'd recommend something along the lines of:

- Using filters to reduce alert fatigue
	- Creating an hourly filter
		- Using sensuctl to create a filter
	        - Assigning the filter to a handler
	- Creating a fatigue check filter
         	- Using a filter asset
		- Annotating a check for filter asset use
		- Assigning the filter to a handler
	- Validating the filter

Don't forget that all headings should use sentence case instead of title case! (Reference: [style guide](https://docs.google.com/document/d/17ALDou5R1MicayMVT4Ew_hIgnkO-RYiMhcAPrCPPtRY/edit#heading=h.bjzkvlsfdt8i))